dist/performance.js

```diff
@@ -264,7 +264,7 @@ function getLcpPreloadInfo(rawDoc, lcpUrl) {
 
         let src = link.href;
         if (link.hasAttribute('imagesrcset')) {
-            src = splitSrcSet(link.imagesrcset).find(src => src == lcpUrl);
+            src = splitSrcSet(link.imagesrcset || link.getAttribute('imagesrcset'))?.find(src => src == lcpUrl);
```

Comment:
Actually I've found the bug!!! That's to [Jake's amazing post](https://jakearchibald.com/2024/attributes-vs-properties/) (thank you @jakearchibald!). The property is case sensitive and has the incorrect casing.

So this works (with optional chaining _just_ in case):

```suggestion
            src = splitSrcSet(link.imageSrcset)?.find(src => src == lcpUrl);
```

Reply:
Though maybe get attribute is safer in that case as not 100% obvious in case someone corrects it later for some reason? But don't think we need both:

```suggestion
            src = splitSrcSet(ink.getAttribute('imagesrcset'))?.find(src => src == lcpUrl);
```

Reply:
LOL that casing is so unintuitive! +1 to using the attribute